root/etc/default/discovery

```diff
@@ -1,4 +1,4 @@
 PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/opt/extension/bin
 FACTERLIB=/usr/share/fdi/facts:/opt/extension/facts
 LD_LIBRARY_PATH=/opt/extension/lib
-RUBYLIB=/opt/extension/lib/ruby
+RUBYLIB=/opt/rh/rh-ruby27/root/usr/share/ruby/vendor_ruby/
```

Comment:
This looks like an SCL path but on EL8 there are no SCLs. Due to modularity everything is in `/usr` under regular paths.

Reply:
Good find, fixed.